src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -1,33 +1,34 @@
 package mate.academy.hibernate.relations;
 
 import java.util.List;
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
+import mate.academy.hibernate.relations.dao.impl.CountryDaoImpl;
+import mate.academy.hibernate.relations.dao.impl.MovieDaoImpl;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.model.Country;
 import mate.academy.hibernate.relations.model.Movie;
-import mate.academy.hibernate.relations.service.ActorService;
-import mate.academy.hibernate.relations.service.CountryService;
-import mate.academy.hibernate.relations.service.MovieService;
+import mate.academy.hibernate.relations.service.impl.ActorServiceImpl;
+import mate.academy.hibernate.relations.service.impl.CountryServiceImpl;
+import mate.academy.hibernate.relations.service.impl.MovieServiceImpl;
 import mate.academy.hibernate.relations.util.HibernateUtil;
 import org.hibernate.SessionFactory;
 
 public class Main {
     public static void main(String[] args) {
-        // use this session factory when you will initialize service instances
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
+        CountryServiceImpl countryService = new CountryServiceImpl(
+                new CountryDaoImpl(sessionFactory));
+        ActorServiceImpl actorService = new ActorServiceImpl(new ActorDaoImpl(sessionFactory));
 
-        Country usa = new Country("USA");
-        CountryService countryService = null; // TODO: initialize this instance
-        countryService.add(usa);
-
+        Country usa = countryService.add(new Country("USA"));
         Actor vinDiesel = new Actor("Vin Diesel");
         vinDiesel.setCountry(usa);
-        ActorService actorService = null; // TODO: initialize this instance
-        actorService.add(vinDiesel);
-
+        vinDiesel = actorService.add(vinDiesel);
+        MovieServiceImpl movieService = new MovieServiceImpl(new MovieDaoImpl(sessionFactory));
```

Comment:
Consider initializing `MovieServiceImpl` in the same manner as `CountryServiceImpl` and `ActorServiceImpl` for consistency. This means using a separate line for the `new MovieDaoImpl(sessionFactory)` instantiation, similar to how it's done for `CountryServiceImpl` and `ActorServiceImpl`.